apollo-runtime/src/main/java/com/apollographql/apollo/subscription/AppSyncOperationMessageSerializer.kt

```diff
@@ -0,0 +1,81 @@
+package com.apollographql.apollo.subscription
+
+import com.apollographql.apollo.api.internal.json.JsonWriter
+import com.apollographql.apollo.api.internal.json.Utils
+import com.apollographql.apollo.api.internal.json.writeObject
+import com.apollographql.apollo.subscription.ApolloOperationMessageSerializer.writePayloadContentsTo
+import okhttp3.HttpUrl
+import okio.Buffer
+import okio.BufferedSink
+import java.util.Base64
+
+class AppSyncOperationMessageSerializer(
```

Comment:
Any reason this is a class and not an object like `ApolloOperationMessageSerializer` ?

Reply:
Damn, silly me 👍